pkg/cachemanager/cachemanager.go

```diff
@@ -193,6 +188,15 @@ func (c *CacheManager) ExcludeProcesses(newExcluder *process.Excluder) {
 	c.excluderChanged = true
 }
 
+// DoForEach runs fn function for each GVK that is being watched by the cache manager.
```

Comment:
A bit more context on why this is necessary would be nice here.

Reply:
added a how to use blurb; let me know if it still needs more context.